### PR TITLE
adding rules

### PR DESCRIPTION
**Pull request by laurafitzgerald**
Adding the rules as discussed
@lgriffin can you take a look at the rules in particular. Let me know if there is something else that should be added in.

### CODE REVIEW

File: first-day/building/index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+        <li>We have a canteen area and most people bring in their own lunch. We discourage leaving the office.</li>
+
+        <li>We clean up after ourselves, cups / plates etc in the dishwasher, rubbish in the bin</li>
+
+        <li>We have free tea/coffee/snacks program which everyone is respectful of and we do not overly abuse it. It's not a substitute for lunch.</li> 
+
+        <li>The rules that you adhere to in school extend to the workplace e.g. profanity, noise level, have respect for peers and colleagues. </li>
+
+        <li>We are customer facing at times and a zero tolerance approach will be taken for any incidents that bring your school or Red Hat into disrepute</li>
```
Comment (damienomurchu) [inline first-day/building/index.html:138]: Would look to add the following:

- note about hardware
- note about the use of internet
- note use of meeting rooms for more vocal discussion/ availability of whiteboards

Comment (lgriffin) [reply first-day/building/index.html:138]: +1

The hardware will be shared, don't grow attached to it. If you have a laptop yourself feel free to bring it in

Internet is open by default but whatever you do will be logged. We won't tolerate any illegal activity on our network and questionable sites are not.

Third point Damien makes is in reference to the fact this is a workplace, respect others and use the meeting rooms and keep noise to a minimum

Comment (laurafitzgerald) [reply first-day/building/index.html:138]: @lgriffin @damienomurchu updated in next commit.

**Review by JameelB — APPROVED** [review]

**Comment by JameelB** [comment]
Verified locally

**Pull request merged**